scripts/src/java/org/oppia/android/scripts/ci/ComputeAffectedTests.kt

```diff
@@ -103,10 +103,18 @@ private fun computeAffectedTargetsForNonDevelopBranch(
   println("Affected test targets due to transitive build deps: $transitiveTestTargets")
 
   val allAffectedTestTargets = (affectedTestTargets + transitiveTestTargets).toSet()
+
+  // Filtering out the targets to be ignored.
+  val filteredAffectedTestTargets = allAffectedTestTargets.filter { targetPath ->
```

Comment:
Suggest alternate name: ``nonInstrumentationAffectedTestTargets`` (better to be specific on what 'filtered' means).

Reply:
Done